frontend/src/components/NavigationButtons.tsx

```diff
@@ -0,0 +1,32 @@
+
+const NavigationButtons = ({ goBack, goNext }) => {
```

Comment:
why did you guys leave files from another issue in your PR? If you have dependency issues, its not a good idea to just simply clone someone else's working branch and then work off of that, leaving all of their work in your working branch when you make a PR. This code has nothing to do with the task at hand. Please remove.